src/main/java/com/hivemq/extension/mqtt/message/log/initializer/Enterprise43Initializer.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2020 dc-square GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.hivemq.extension.mqtt.message.log.initializer;
+
+import com.hivemq.extension.mqtt.message.log.config.MqttMessageLogConfig;
+import com.hivemq.extension.sdk.api.annotations.NotNull;
+import com.hivemq.extension.sdk.api.client.ClientContext;
+import com.hivemq.extension.sdk.api.client.parameter.InitializerInput;
+import com.hivemq.extension.sdk.api.services.Services;
+import com.hivemq.extension.sdk.api.services.intializer.ClientInitializer;
+
+import static com.hivemq.extension.mqtt.message.log.interceptor.InterceptorUtil.*;
+
+/**
+ * Creates a {@link ClientInitializer} that is usable for any HiveMQ 4.3 Enterprise Version.
+ *
+ * @author Michael Walter
+ * @version 1.1.0
+ */
+public class Enterprise43Initializer implements ClientInitializer {
```

Comment:
where is the difference between this and the communityInitializer?

Reply:
When I was refactoring the code it made sense, but looking over it now you are right this can be merged with the CommunityInitializer.

My question is how should I name the combined Initializer? Enterprise43AndCommunityInitializer?

Reply:
Done